src/components/player/content/index.js

```diff
@@ -18,6 +20,10 @@ const Content = ({
 }) => {
   if (layout.single) return null;
 
+  const isTldrawWhiteboard = storage.tldraw.length ||
+                             storage.panzooms.tldraw ||
+                             storage.cursor.tldraw;
```

Comment:
What does this means?
- Why is there `tldraw` inside `panzooms` and `cursor`?
- Why testing `tldraw` isn't enough?